sputnikdao2/Cargo.toml

```diff
@@ -18,14 +18,16 @@ hex = "0.4.3"
 version = "1.4.0"
 
 [dev-dependencies]
-near-workspaces = "0.17.0"
+near-sandbox = { git = "https://github.com/near/near-sandbox-rs.git", branch = "sandbox-rpc" }
+near-api = { version = "0.7", default-features = false }
 test-token = { path = "../test-token" }
 sputnik-staking = { path = "../sputnik-staking" }
 sputnikdao-factory2 = { path = "../sputnikdao-factory2" }
 tokio = { version = "1.44.0", features = ["full"] }
 walrus = "0.23.3"
 anyhow = "1.0.97"
 rand = "0.8"
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Why do we need openssl here? We don't even use HTTPS for local testing

Reply:
I believe, it's only on my nixos setup :)